openedx/core/djangoapps/user_authn/views/registration_form.py

```diff
@@ -348,14 +348,20 @@ def __init__(self):
             handler = getattr(self, "_add_{field_name}_field".format(field_name=field_name))
             self.field_handlers[field_name] = handler
 
+        custom_form = get_registration_extension_form()
+        if custom_form:
+            custom_form_field_names = [field_name for field_name, field in custom_form.fields.items()]
```

Comment:
I think we could have done something like this here
`custom_form_field_names =  list(custom_form.fields.keys())`
what do you think?

Reply:
I agree, this could be simplified, though it is already merged on edX master and possibly I wouldn't make an unnecessary code drift.

Reply:
Sure! sounds reasonable.